aerospike/samples/avs_bedrock_integration.ipynb

```diff
@@ -0,0 +1,400 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "IQt5lMKvxios"
+      },
+      "source": [
+        "# [Aerospike Vector Store(AVS)](https://aerospike.com/docs/vector) demo with [Amazon Bedrock](https://aws.amazon.com/bedrock/)\n",
```

Comment:
We aren't here really explaining what this demo is going to show (namely an example of switching models) Are we still seeing that as the main advantage with bedrock (+aerospike)?

Reply:
The notebook, on the top states that we are analysing text using mentioned amazon text models. It also states that it uses AVS for vector storage and retrieval. What other information you think should be added? @arrowplum